diff --git a/node_modules/hast-util-to-string/lib/index.js b/node_modules/hast-util-to-string/lib/index.js
index 17fc432..7427395 100644
--- a/node_modules/hast-util-to-string/lib/index.js
+++ b/node_modules/hast-util-to-string/lib/index.js
@@ -19,7 +19,7 @@ export function toString(node) {
   }
 
   // “Context object’s data.”
-  return 'value' in node ? node.value : ''
+  return 'value' in node ? handleRawWithImgTag(node) : ''
 }
 
 /**
@@ -31,6 +31,8 @@ export function toString(node) {
 function one(node) {
   if (node.type === 'text') {
     return node.value
+  } else if (node.type === 'raw') {
+    return handleRawWithImgTag(node)
   }
 
   return 'children' in node ? all(node) : ''
@@ -46,6 +48,10 @@ function all(node) {
   let index = -1
   /** @type {Array<string>} */
   const result = []
+  if ('value' in node) {
+    result.push(node.value)
+    index++
+  }
 
   while (++index < node.children.length) {
     result[index] = one(node.children[index])
@@ -53,3 +59,12 @@ function all(node) {
 
   return result.join('')
 }
+
+function handleRawWithImgTag(node) {
+  if (node.value.indexOf('<img')>=0) {
+    // NOTE : twemoji creates a raw node with an img tag inside.
+    // here, I extract the alt to use as text.
+    return node.value.replace(/\<img.+ alt=\"(.+?)\" .+\>/, '$1')
+  }
+  return node.value
+}
